catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/GetParametersRequestImpl.java

```diff
@@ -141,23 +141,23 @@ public State complete_StringDAG(StringDAGHolder parameters) throws ProcessingFau
   }
 
   @Override
-  public RequestDescription get_request_description() throws ProcessingFault, SystemFault {
+  public RequestDescription getRequestDescription() throws ProcessingFault, SystemFault {
```

Comment:
❗️ You can't rename these methods. They override interface methods in generated CORBA api code.